.github/workflows/build_images.yml

```diff
@@ -0,0 +1,61 @@
+name: Build docker images
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Needs to skip if github_branch != nextcloud-releases
See https://docs.github.com/en/actions/learn-github-actions/environment-variables

Reply:
not needed anymore